src/components/FormComponent/FormComponent.css

```diff
@@ -0,0 +1,64 @@
+.form-style {
+  font: 14px Arial, Helvetica, sans-serif;
+  max-width: 400px;
+  margin: 10px auto;
+  padding: 16px;
+  background: #F7F7F7;
+}
+
+.form-input {
```

Comment:
This should be in separate css file, containing only css related to InputFieldComponent